tools/local-dev.sh

```diff
@@ -2,31 +2,50 @@
 
 set -e
 ## This script sets up the environment for local development.
-## Dependencies: docker, chmod
+## TERRA_CLI_DOCKER_MODE environment variable controls docker support. Set to
+##     DOCKER_NOT_AVAILABLE (default) to skip pulling the Docker image
+#      or DOCKER_AVAILABLE to pull the image (requires Docker to be installed and running).
```

Comment:
ditto above: Please add notes about this to the `CONTRIBUTING.md` in the Setup development environment section.